tests/Integration/bootstrap.php

```diff
@@ -30,6 +30,16 @@
 require_once "{$_tests_dir}/includes/functions.php";
 require_once __DIR__ . '/../../vendor/yoast/wp-test-utils/src/WPIntegration/bootstrap-functions.php';
 
+// Define required constants for testing.
+$test_ws_url = getenv( 'TEST_VIP_RTC_WS_URL' );
+if ( false !== $test_ws_url ) {
+	// If the environment variable is set (even if empty), use its value.
+	define( 'VIP_RTC_WS_URL', $test_ws_url );
+} else {
+	define( 'VIP_RTC_WS_URL', 'ws://localhost:1234' );
+}
```

Comment:
What is the point of this code? Are there tests that depend on a varying definition of `VIP_RTC_WS_URL`? Why not hardcode a test value here, similar to `VIP_RTC_WS_AUTH_SECRET` below?